### DISCUSSION TITLE
Chakra UI with Angular ✌️😁

### DISCUSSION THREAD

**Discussion by t1m4lc** [chakra-ui/chakra-ui#6086] [General]
Hello everyone,
I would like to develop this ui kit with angular. Who's up for that?
However where do I find the basic style to have the same style and the same utility class for all the components?
Thanks see you soon

**Comment by aarhusgregersen** [comment]
I'd be down to help, contributing as much or as little as I can. 😄

**Reply by sefatanam** [reply]
looking forward to it....

**Reply by nutangwari** [reply]
any updates or link to the repo if this was started and need more contributors.

**Comment by HadySalhab** [comment]
Any updates? I'd like to develop/contribute this ui in angular too. 
If a repo already exists, can anyone add the link?

**Comment by shadhin08** [comment]
Any updates in there?

**Comment by pociej** [comment]
HI guys any progress? I opened similar, more general discussion here https://github.com/chakra-ui/chakra-ui/discussions/8210

**Comment by abingineer** [comment]
Hello guys any progress about chakra integration within angular ??

**Comment by andresh0816** [comment]
Hey, I can contribute to bring this beautiful UI kit to Angular. If there's repository or css base to this just tell us, to start de development.

**Comment by pehcst** [comment]
Any updates?
